api/clan_test.go

```diff
@@ -655,6 +655,190 @@ var _ = Describe("Clan API Handler", func() {
 
 			Expect(result["roster"] == nil).To(BeFalse())
 		})
+
+		It("Should fail with 400 if maxPendingApplications cannot be parsed as uint", func() {
+			_, clan, _, _, _, err := models.GetClanWithMemberships(testDb, 0, 0, 0, 0, "", "")
```

Comment:
😮 I know this is only _test and the code is already merged, but 6 is toooo much to return from a func.

Reply:
(I don't know if it's worth changing it now, just a comment)